### PR TITLE
[Flutter Parent][MBL-13015] Syllabus Screen

### PR DESCRIPTION
**Pull request by StephenBrough**
Testing:
	Mobile dev has a course with a syllabus that has embedded media and links, good for testing.

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1018947&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/flutter_parent/lib/screens/courses/details/course_syllabus_screen.dart [hunk 1]
```diff
@@ -12,21 +12,145 @@
...
+            Uri.dataFromString(syllabusWrapper.replaceAll('{\$CONTENT\$}', syllabus),
+                    mimeType: 'text/html', encoding: Encoding.getByName('utf-8'))
+                .toString(),
+          );
+        },
       ),
     );
   }
+
+  // This HTML wrapper makes large media content scale to fit the screen
```
Comment (Blackchai) [inline apps/flutter_parent/lib/screens/courses/details/course_syllabus_screen.dart]: You can place this into an html file in the assets directory, and pull that in for use. If you want to leave it here for now, I can change this in my PR so we can just do it in one place :).

Comment (StephenBrough) [reply apps/flutter_parent/lib/screens/courses/details/course_syllabus_screen.dart]: 👍 Let's get that taken care of in your upcoming PR

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1018960&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1019280&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Pull request merged**